src/pages/index.js

```diff
@@ -66,40 +128,82 @@ profileEditButton.addEventListener("click", () => {
   editPopupForm.open();
 });
 
-// Function to render a card
+avatarEditButton.addEventListener("click", () => {
+  avatarEditPopup.open();
```

Comment:
You need to call `toggleButtonState` for the avatar form as well to disable the button